jcl/src/openj9.criu/share/classes/org/eclipse/openj9/criu/CRIUSupport.java

```diff
@@ -203,6 +203,16 @@ public static boolean isCRIUSupportEnabled() {
 		return (nativeLoaded && InternalCRIUSupport.isCRIUSupportEnabled());
 	}
 
+	/**
+	 * Queries if CRIU Checkpoint is allowed. With -XX:-CRIUSecProvider
+	 * requires full providers back.
+	 *
+	 * @return TRUE if support is enabled and the library is loaded, FALSE otherwise
+	 */
```

Comment:
I think this comment can be improved too; suggest
```java
	/**
	 * Queries if the CRIUSecProvider should be disabled even if CRIU
	 * checkpoints are allowed (checks whether -XX:-CRIUSecProvider
	 * has been specified).
	 *
	 * @return true if CRIUSecProvider is disabled, otherwise false
	 */
```

Reply:
Updated

Reply:
This javadoc is stale as well.

Reply:
Updated